src/Rules/StarterWeb/IndividualAuth/Controllers/AccountController.cs

```diff
@@ -160,8 +160,13 @@ public IActionResult ExternalLogin(string provider, string returnUrl = null)
         // GET: /Account/ExternalLoginCallback
         [HttpGet]
         [AllowAnonymous]
-        public async Task<IActionResult> ExternalLoginCallback(string returnUrl = null)
+        public async Task<IActionResult> ExternalLoginCallback(string returnUrl = null, string remoteError = null)
         {
+            if (remoteError != null)
+            {
+                ModelState.AddModelError("", $"Error from external provider: {remoteError}");
```

Comment:
Use string.Empty instead of "". This is the pattern in the AccountController towards the end of file